### PR TITLE
Fix release workflow

### PR DESCRIPTION
**Pull request by Adamantios**
## Description
Uses the docker user as an image author and also tags with `latest`.

## More information
This PR fixes an issue with the release workflow and also tags with `latest` which was missing.

#### Issue
Previously, we were using the default image author, which is the IPFS author, which is initialized using the author of the service package, i.e., `elcollectooorr`. The release workflow was failing because we were instead trying to push the image using the docker user as an image author.

#### Resolution
By introducing this change, we now use the docker user as the image author when building the image.

#### Suggestion
Please note that we might want to consider keeping the old approach when building the image, i.e., using the package author as the image author. In this case, we would push using this image name instead of the docker user. This might be the best approach as the image's name would contain the author of the package.

However, if we were to keep the current PR's approach it might also make more sense to rename the `elcollectooorr` author of the packages.

### CODE REVIEW

File: .github/workflows/release.yaml
Comment (0xArdi) [inline .github/workflows/release.yaml:83]: Is this necessary?
Also the image we are building is the agent image. In this case it shouldnt be a problem because the service name matches the agent name.

Comment (Adamantios) [reply .github/workflows/release.yaml:83]: > Is this necessary?

Do you mean tagging with `latest`? No, it is not necessary just preferrable.

> Also the image we are building is the agent image. In this case it shouldnt be a problem because the service name matches the agent name.

Yes, this is correct. I just followed the same approach as previously. For example, if you check the deleted code below, you will notice that the service was being used. e.g.:

```
echo "Pushing $DOCKER_USER/$SERVICE:$VERSION"
```

We could add a:

```
grep 'agent' packages/packages.json | awk -F/ '{print $3}' | head -1
```

for the agent. What do you think?

**Review by 0xArdi — APPROVED** [review]

File: .github/workflows/release.yaml
Comment (8ball030) [inline .github/workflows/release.yaml:97]: Are you sure this works without the ;

See above comment

File: .github/workflows/release.yaml
Comment (Adamantios) [reply .github/workflows/release.yaml:84]: This will tag the image created above with the version, e.g., `v0.10.0` and with `latest`.

File: .github/workflows/release.yaml
Comment (8ball030) [reply .github/workflows/release.yaml:97]: Yeah that looks good, would recommend you make your test bash.sh script much more comprehensive to ensure it functions correctly with the expected set env vars. 
I.e. doe you get the images tagged you expect that align with the output of open autonomy deploy build

Comment (Adamantios) [reply .github/workflows/release.yaml:97]: As far as I can tell from testing it, it works as expected.

Comment (8ball030) [reply .github/workflows/release.yaml:97]: we have multiple times had issues with this.
Can you please execute the code and provide proof of this, so we dont waste more time.

Comment (8ball030) [reply .github/workflows/release.yaml:97]: It is however of course your repo, but i will not approve without this, feel free to find another reviewer :)

File: .github/workflows/release.yaml
Comment (Adamantios) [inline .github/workflows/release.yaml:22]: The `latest` tag was causing issues locally when trying to test the workflow as an older version of the image was being used.

Better to explicitly tag and update every time we bump.

File: .github/workflows/release.yaml
Comment (8ball030) [reply .github/workflows/release.yaml:98]: Lgtm thanks friend

**Pull request merged**
